assets/css/gutenberg-syndicated-post.scss

```diff
@@ -17,6 +17,7 @@ body.dt-linked-post {
 		cursor: default;
 	}
 
+	.edit-post-welcome-guide,
```

Comment:
This change is to allow `cy.closeWelcomeGuide();` to run by re-enabling pointer events on the modal. Without them, Cypress declines to click on the element because it's unclickable.